### DISCUSSION TITLE
News Cache [2023-10-16]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#4494] [Announcements]
### 🚢  Ships

- [Vercel Postgres is now generally available for Hobby and Pro users](https://vercel.com/changelog/vercel-postgres-is-now-available-for-pro-users)
- [Strengthening Vercel's Infrastructure against HTTP/2 Rapid Reset Attacks](https://vercel.com/changelog/strengthening-vercels-infrastructure-against-http-2-rapid-reset-attacks)
- [Announcing v0: Generative UI](https://vercel.com/blog/announcing-v0-generative-ui)

### 💡 Inspiration

- [How we optimized package imports in Next.js](https://vercel.com/blog/how-we-optimized-package-imports-in-next-js)
- [Tekla's ecommerce evolution: harnessing flexibility with Vercel and Medusa](https://vercel.com/customers/teklas-ecommerce-evolution-harnessing-flexibility-with-vercel-and-medusa)

### 📚 Education

- [Forrester Trends Report: Understand developer experience to improve business outcome](https://vercel.com/blog/forrester-analyst-report-understand-developer-experience-to-improve-business)
- [Next.js Conf 2023 speakers](https://nextjs.org/conf/speakers?v=23)

**Comment by Aeshtech** [comment]
Can you please help related to build issue ? 
 
20:03:54.739  Finished
20:03:54.741  Note: No functions dir at /functions found. Skipping.
20:03:54.741  Validating asset output directory
20:03:54.742  Error: Output directory ".vercel/output/static" not found.
20:03:55.777  Failed: build output directory not found


This is the build error coming on pushing next js changes to cloudflare.
So I was reverted back also to check if something not working to current commit but still same issue. 

I have also deleted server cache, on local everything working fine but build is failed on cloudflare.

**Reply by amyegan** [reply]
Please [start a new discussion](https://github.com/vercel/community/discussions/new) with more information about the problem, steps we can follow to reproduce the error, and a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that can be used to help with debugging 🙏
